### ISSUE TITLE
Correct way to get ReCoRD logits?

### ISSUE THREAD

**Issue by ethanjperez** [rrmenon10/ADAPET#16]
Is it possible to get the ReCoRD prediction logits, probabilities, or logprobs on dev? I see in [this line](https://github.com/rrmenon10/ADAPET/blob/master/src/eval/Scorer.py#L213) that there is a hack that just returns zeros for the logits -- Is there an easy way to get the logits, maybe using the logits from this [line](https://github.com/rrmenon10/ADAPET/blob/master/src/eval/eval_model.py#L21)? (Assuming it is possible to treat those as logits)

**Comment by rrmenon10** [comment]
You're right, we can use the line you've mentioned to get the logits. However, we did not use it (as logits to compute probabilities) since the number of candidate answers was variable for ReCoRD. We wanted to have a simple batched implementation of the method.
